pyomo/contrib/pyros/tests/test_uncertainty_sets.py

```diff
@@ -2623,6 +3050,88 @@ def test_compute_parameter_bounds(self):
         self.assertEqual(custom_set._compute_parameter_bounds(baron), [(-1, 1)] * 2)
 
     @unittest.skipUnless(baron_available, "BARON is not available")
+    def test_solve_feasibility(self):
+        """
+        Test uncertainty set feasibility problem gives expected results.
+        """
+        # feasibility problem passes
+        baron = SolverFactory("baron")
+        custom_set = CustomUncertaintySet(dim=2)
+        self.assertTrue(custom_set._solve_feasibility(baron))
+
+        # feasibility problem fails
+        custom_set.parameter_bounds = [[1, 2], [3, 4]]
+        exc_str = r"Could not successfully solve feasibility problem. .*"
+        with self.assertRaisesRegex(ValueError, exc_str):
+            custom_set._solve_feasibility(baron)
+
+    # test default is_bounded
+    @unittest.skipUnless(baron_available, "BARON is not available")
+    def test_is_bounded(self):
+        """
+        Test boundedness check computations give expected results.
+        """
+        custom_set = CustomUncertaintySet(dim=2)
+        CONFIG = pyros_config()
+        CONFIG.global_solver = global_solver
+
+        # using provided parameter_bounds
+        start = time.time()
+        self.assertTrue(custom_set.is_bounded(config=CONFIG), "Set is not bounded")
+        end = time.time()
+        time_with_bounds_provided = end - start
+
+        # when parameter_bounds is not available
+        custom_set.parameter_bounds = None
+        start = time.time()
+        self.assertTrue(custom_set.is_bounded(config=CONFIG), "Set is not bounded")
+        end = time.time()
+        time_without_bounds_provided = end - start
+
+        # check with parameter_bounds should always take less time than solving 2N
+        # optimization problems
+        self.assertLess(
+            time_with_bounds_provided,
+            time_without_bounds_provided,
+            "Boundedness check with provided parameter_bounds took longer than expected.",
```

Comment:
Wrap this string to reduce the line length

```suggestion
            "Boundedness check with provided parameter_bounds took longer than "
            "expected.",
```

Reply:
This time-based test has been removed